src/ui/style.css

```diff
@@ -72,6 +72,8 @@ h1,h2,h3,h4,h5,h6 {
   * Next Event
   */
 .event {
+  max-width: 370px;
```

Comment:
The element should be flexible - constrain the width of it temporarily in the demo html with a `<div style="width:400px">`